core-bundle/src/Resources/contao/config/config.php

```diff
@@ -282,11 +282,6 @@
 (
 	'tables' => array
 	(
-		'index' => array
-		(
-			'callback' => array('Contao\Automator', 'purgeSearchTables'),
-			'affected' => array('tl_search', 'tl_search_index')
```

Comment:
I understand why you have removed this, but being able to see the number of entires in the tl_search tables was pretty helpful. Are you planning on re-adding it in some way in your crawler implementation?

Reply:
We would need to implement some `getNumberOfEntries()` on the `IndexerInterface`. Not sure if that makes sense?

Reply:
I am not sure if it makes sense on the interface, either.

I only know that being able to see the number of entries in the tl_search tables helped me see whether rebuilding the search index was successful and it helped me spot problems that caused the search index to grow too much. Therefore I would like to keep this information visible somehow.

Reply:
Okay, I see. Good point. We do need to extend the interface then. Otherwise, if I disable the default search indexer and implement some Algolia search, this information is not helpful here, right?
I don't think you need to know the entries of tl_search and tl_search_index. A normal user doesn't know what these two tables are for anyway, right? I guess you only need them to see if there are any entries or not, correct?

Reply:
Yes.

Reply:
Okay, let me come up with a proposal then :)